corehq/apps/app_manager/templates/app_manager/partials/forms/case_config_advanced.html

```diff
@@ -190,13 +190,18 @@ <h4 class="panel-title panel-title-nolink">
 
 <script type="text/html" id="case-config:case-action">
     <div class="panel-heading clickable panel-heading-case-action" data-toggle="collapse"
-         data-bind="attr: {href: '#' + actionType + $index()}">
+         data-bind="attr: {href: '#' + actionType + $index()}, css: {not_droppable: moveable()}">
```

Comment:
nit, classes normally use `-` not `_` (so this would be `'not-droppable'`, with the quotes)

Reply:
Nice. I didn't think about using quotes to get around knockout syntax errors. https://github.com/dimagi/commcare-hq/pull/23689/commits/adb632b00d1d84ed4211e6a5a32111e4f2c353c8